### PR TITLE
Amplitude

### PR DESCRIPTION
**Pull request by liamcardenas**

### CODE REVIEW

File: src/renderer/js/main.js [hunk 1]
```diff
@@ -66,18 +89,29 @@ var init = function() {
   app.store.dispatch(doDownloadLanguages());
 
   function onDaemonReady() {
-    window.sessionStorage.setItem("loaded", "y"); //once we've made it here once per session, we don't need to show splash again
-    app.store.dispatch(doDaemonReady());
+    lbry.status().then(info => {
+      amplitude
+        .getInstance()
+        .init(
+          "0b130efdcbdbf86ec2f7f9eff354033e",
```
Comment (seanyesmunt) [inline src/renderer/js/main.js]: what is this value?

File: src/renderer/js/main.js [hunk 2]
```diff
@@ -48,6 +50,27 @@ ipcRenderer.on("window-is-focused", (event, data) => {
...
+          .getInstance()
+          .logEvent("CLICK", {
+            target: element,
+            location:
+              hrefParts.length > 1 ? hrefParts[hrefParts.length - 1] : "/",
+          });
+      } else {
+        amplitude
+          .getInstance()
+          .logEvent("UNMARKED_CLICK", {
```
Comment (seanyesmunt) [inline src/renderer/js/main.js]: Do you have an example of when this would fire? Just curious

Comment (liamcardenas) [reply src/renderer/js/main.js]: For instance, if there is an icon with no title attribute. I put this in there so, if we see it, we can go and add a title.

File: src/renderer/js/main.js (hunk 1 above)
Comment (liamcardenas) [reply src/renderer/js/main.js]: this is our (public) api key

File: src/renderer/js/main.js (hunk 2 above)
Comment (seanyesmunt) [reply src/renderer/js/main.js]: 👍

**Pull request merged**